### ISSUE TITLE
Installed unlocker, Confused now ;-;

### ISSUE THREAD

**Issue by BiscItz** [lanylow/rbxfpsunlocker-osx#289]
Ok so, im rlly new here so im just gonna say my entire story

So basically i was playing tka and i asked my friend how he was so fast, he said fps unlocker, he sent me this link and i followed the instructions that were in the main page but now im kinda clueless on what to do if its automatic or do i do extra stuff so help would be nice

if u need extra info just tell me i can provide any info needed

(btw im new to github too)

**Comment by lanylow** [comment]
Yes, after following the instructions and installing the unlocker everything should just work. You can check your frame rate in Roblox by clicking Shift + F5 or Option + Shift + F5. Assuming your mac is powerful enough, you should see more than 60 FPS.
